app/keepers/keepers.go

```diff
@@ -606,11 +605,6 @@ func NewAppKeepers(
 		),
 	)
 
-	// register wasm gov proposal types
-	// The gov proposal types can be individually enabled
-	if len(enabledProposals) != 0 {
-		govRouter.AddRoute(wasmtypes.RouterKey, wasm.NewWasmProposalHandler(appKeepers.WasmKeeper, enabledProposals)) //nolint:staticcheck // we still need this despite the deprecation of the gov handler
```

Comment:
We don't need this anymore because of sdk47 right?

Reply:
yes, messages are generated & routed properly 😃